src/abi_1_0.c

```diff
@@ -303,6 +311,7 @@ struct fi_info_1_7 {
         struct fi_domain_attr_1_7 *domain_attr;
         struct fi_fabric_attr_1_7 *fabric_attr;
         struct fid_nic_1_7        *nic;
+        struct fi_hmem_attr_1_7   *hmem_attr;
```

Comment:
Changes to this file need to be dropped.  These are definitions for the prior ABI structures.

Reply:
Ack